HHS/Head-Start-TTADP
# [TTAHUB-1694] Use filter pattern for reportText instead of ElasticSearch

## Description of change

We were previously using Elastic Search to find report IDs that match some query. This PR removes that functionality in favor of using the filter pattern that we've created. It still matches on the same things that ES was indexed on, so results should be basically the same.

## How to test

Apply the report text filter and check your backend logs. Validate that the generated query looks correct to you.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1694


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
